### PR TITLE
Add thumbnail images for Playlab projects

### PR DESCRIPTION
**Pull request by davidsbailey**
Also adds a new project-backed playlab level, so that we can remix it to get a new project in the projects list for use in eyes tests.

The SVG capture process seems a little more CPU intensive than artist or gamelab, so only allow it once per minute.

expected new eyes baselines: https://eyes.applitools.com/app/sessions/00000251909492557021/00000251909492556129/steps/1?&parent=vos3oAR&accountId=mG4DoRfv0ntEgI3XIYBulrKxlZqrHtGwbP3V103ojvLFU110

### CODE REVIEW

File: apps/src/studio/studioThumbnail.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    console.warn('Thumbnail capture failed: svg.toDataURL undefined.');
+    return;
+  }
+
+  svgToDataURI(svg)
+    .then(imageFromURI)
+    .then(canvasFromImage)
+    .then(createThumbnail)
+    .then(canvasToBlob)
+    .then(project.saveThumbnail);
```
Comment (Bjvanminnen) [inline apps/src/studio/studioThumbnail.js]: Would it make sense to move more of this logic into imageUtils? i.e. have something like
```
captureSvgScreenshot(document.getElementById('svgStudio'), MIN_CAPTURE_INTERVAL_MS);
```
I suspect we will have other svg apps that could then reuse much of this?

File: dashboard/config/scripts/allthethings.script [hunk 2]
```diff
@@ -103,7 +104,6 @@ stage 'Pixelation'
 level 'Pixelation - Lesson 14 - Make the Letter A'
 level 'Pixelation - Lesson 15 - Complete 3-bit color'
 level 'Pixelation - Lesson 15 - Color Shades 4x4'
-level 'Pixelation - AllTheThings 4'
```
Comment (Bjvanminnen) [inline dashboard/config/scripts/allthethings.script]: intentionally removing pixelation level?

**Comment by davidsbailey** [comment]
Good suggestion Brent. PTAL. There is more cleanup I can do affecting other app types, but it probably warrants a follow-on PR.

**Comment by Bjvanminnen** [comment]
lgtm

**Pull request merged**